src/smtlibparser.yy

```diff
@@ -268,7 +268,7 @@ term_s_expr:
          $$ = drv.solver()->make_term(po, *$3);
        }
     }
-    else if (uf = drv.lookup_symbol($2))
```

Comment:
Just a question for my understanding, is there a reason for the extra set of parentheses here or is that just a style thing?

Reply:
So this, and a couple of other unrelated diffs appear in this PR for some reason, even though they were already merged by Ahmed in his PR:
https://github.com/stanford-centaur/smt-switch/commit/53265962d835ecf88a9f59bd6e70c18f7056cdd8

The reason he added these was compiler warnings.